.github/workflows/windows-build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_SYSTEM_VERSION=10.0.22621.0 -DPPX_BUILD_TESTS=ON ..
```

Comment:
If is duplicating the workflow a lot of work? Would probably be better to have both path tested, since here we allow the non-XR path to rot on windows (some failure could be caught by linux build of course, but wonder if it's expensive to duplicate the build)

Reply:
There's a cost in terms of CI resources. The more we add, the more resources are used and depending on the number of concurrent PRs, it can slow down the CI. That said, we're pretty lean on usage right now so it makes sense to add a separate path, but let's do it in a separate PR.